etc/rose-rug-brief-tour/suite.rc

```diff
@@ -30,22 +26,18 @@
 
 [runtime]
     [[root]]
-        command scripting = rose task-run -v
-        pre-command scripting = {{SLEEP}}  # Slow down so we can see it run
+        script = rose task-run -v
+        pre-script = {{SLEEP}}  # Slow down so we can see it run
         [[[remote]]]
             host = {{ ROSE_ORIG_HOST }}
         # Begin Rose recommended event hook block
         [[[event hooks]]]
-           succeeded handler = rose suite-hook
-           failed handler = rose suite-hook --shutdown
-           submission failed handler = rose suite-hook --shutdown
-           submission timeout handler = rose suite-hook --mail
-           execution timeout handler = rose suite-hook --mail
-           submission timeout = P1D # 24 hours
-           execution timeout = PT3H # 3 hours
+            mail events = submission timeout, execution timeout
```

Comment:
I thought that only covered actual task failure not submission failure? If it does now cover both, then can we no longer disambiguate the two?